### ISSUE TITLE
Kibble showing up always.

### ISSUE THREAD

**Issue by Pyr088** [thommcgrath/Beacon#28]
I had kibble dropping but it was too high. I've changed it and im still getting HIGH amounts of kibble dropping in white drops. All kinds of kibble as well. So i've made another beacon file deleted all the kibble in the drops completely and exported it and it is still showing kibble coming in the white drops in the .ini file. 
[Nokibble.zip](https://github.com/thommcgrath/Beacon/files/782833/Nokibble.zip)

Maybe im doing something wrong but i cant see what if i am.

**Comment by thommcgrath** [comment]
I can confirm the behavior. That's really weird. I'll look into it ASAP.

**Comment by Pyr088** [comment]
I made a completely new document and copied everything but the kibble set over for every drop. It did not add kibble then. I then copied the kibble over and it was correct only having 1-2 kibble per drop instead of sometimes 60 in the older broken ini. I think it has something to do with it keeping a memory and not actually deleting a set when you tell it too. Just my two cents.

**Comment by thommcgrath** [comment]
The cause is related to pasting an item set. If a loot source already has an identical set, Beacon gets confused. The two sets become essentially hidden, thanks to the way Beacon shows only item sets common to all selected sources.

So, for example, if you have a single loot source selected, the filter logic looks for item sets with exactly 1 match. Simple, of course. With two sources selected, Beacon looks for sets with exactly 2 copies. And so on. With the duplicates, if you have 5 duplicates, that doesn't match the 1 count the filter is looking for, so it isn't shown.

Long story short, you found a bug. I'll need to alter how pasting works to make sure duplicated sets can't get into a source.

**Comment by Pyr088** [comment]
Thanks so much!!! Wouldn't even mess with beacons if it wasn't for this tool.
